### PR TITLE
index.html

### PR DESCRIPTION
**Pull request by SMITHsharon**

### CODE REVIEW

**Comment by SMITHsharon** [comment]
@zoeames / or who?
Working on the readme now. 
This quiz is perfectly functional, except for, when you edit a 2nd or nth Car card, the text is mirrored in all prior cards. I have isolated this error down to the exact line where it's happening (selectedDisplay.js, line50), but no matter what I change, this line still "loops" in its execution, as can be seen by the console.log : / 
A rare time when I.Give.Up : (

**Comment by SMITHsharon** [comment]
This is ready for review. 
One question above ^^^. 
cheers ;)

**Comment by zoeames** [comment]
@SMITHsharon go back and look at how you did WYSIWYG

**Comment by SMITHsharon** [comment]
@zoeames 
ditched the solitary *form* tags
The source of hours of #*#*#*#*#*#*#. 
Works like a charm. Ready for review ~sh

File: CarLot.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	// stores the data in the <carInventory> object array;
+	// then calls <populatePage> function to write the data to the DOM
+	oldCarLot.executeThisCodeAfterFileLoaded = function() {
+
+		carInventory = JSON.parse(this.responseText);
+		oldCarLot.populatePage(carInventory);
+	};
+	
+
+	CarLot.executeThisCodeAfterFileFails = function() {
```
Comment (zoeames) [inline CarLot.js]: this should be oldCarLot. executeThisCodeAfterFileFails

File: CarLot.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		console.log("Inventory Load Fail : / ");
+	};
+
+
+	// executes XHR Request
+	CarLot.loadInventory = function() {
```
Comment (zoeames) [inline CarLot.js]: this should be oldCarLot. loadInventory

File: CarLot.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		
+		var inventoryLoader = new XMLHttpRequest();
+
+		inventoryLoader.addEventListener("load", CarLot.executeThisCodeAfterFileLoaded);
```
Comment (zoeames) [inline CarLot.js]: this should be oldCarLot. executeThisCodeAfterFileLoaded

File: CarLot.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+		inventoryLoader.addEventListener("error", CarLot.executeThisCodeAfterFileFails);
```
Comment (zoeames) [inline CarLot.js]: this should be oldCarLot. executeThisCodeAfterFileFails

File: CarLot.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+		inventoryLoader.open("GET", "inventory.json");
+		inventoryLoader.send();	
+	};
+
+
+	// function RETURNs <carInventory> array to whatever function calls for it
+	CarLot.getCarInventory = function() {
```
Comment (zoeames) [inline CarLot.js]: this should be oldCarLot.getCarInventory

File: CarLot.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+			
+		return carInventory;
+	};
+
+	// function writes the edited Car description to <carInventory> object array
+	CarLot.updateCarDescription = function(carID, carDesc) {
```
Comment (zoeames) [inline CarLot.js]: this should be oldCarLot. updateCarDescription

**Comment by zoeames** [comment]
Very nice quiz!!  you are 98% of the way there.  In Carlot.js unless you are calling a function from another augmentor you should have oldCarLot.functionName.

File: CarLot.js (hunk 1 above)
Comment (SMITHsharon) [reply CarLot.js]: done.

File: CarLot.js (hunk 2 above)
Comment (SMITHsharon) [reply CarLot.js]: done 👍

File: CarLot.js (hunk 3 above)
Comment (SMITHsharon) [reply CarLot.js]: done 👍

File: CarLot.js (hunk 4 above)
Comment (SMITHsharon) [reply CarLot.js]: done 👍

File: CarLot.js (hunk 5 above)
Comment (SMITHsharon) [reply CarLot.js]: done 👍

File: CarLot.js (hunk 6 above)
Comment (SMITHsharon) [reply CarLot.js]: done 👍

**Comment by SMITHsharon** [comment]
@zoeames 
Made these changes, but am confused? 
"unless you are calling a function from another augmentor ..."
So now the way mine is written, every function declaration (?/is that the term?) and function call is prefixed with <oldCarLot>, except for the one call that launches the application in <main.js> : CarLot.loadInventory();

All of the changes you have listed here are located in the file CarLot.js.
So even if the function that is being called is located in the same file, it still is referenced oldCarLot?
#confusing

Is this bc I set this up structurally so that all the files are augmentors to (or of?) each other? There's not a /main/ one like we did with SolarSystem.js ?

**Comment by zoeames** [comment]
👍

**Pull request merged**